docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/OrdinaryLeastSquaresAdvanced.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Samples.Dynamic.Trainers.Regression
+{
+    public static class OrdinaryLeastSquaresAdvanced
```

Comment:
>OrdinaryLeastSquaresAdvanced [](start = 24, length = 28)

is there a tt for this?

Reply:
No for the reason to LightGbmAdvanced.cs.

---
In reply to: [275458236](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275458236) [](ancestors = 275458236)